.github/helper/install.sh

```diff
@@ -40,14 +40,16 @@ sed -i 's/schedule:/# schedule:/g' Procfile
 sed -i 's/socketio:/# socketio:/g' Procfile
 sed -i 's/redis_socketio:/# redis_socketio:/g' Procfile
 
-bench get-app payments --resolve-deps
-bench get-app https://github.com/frappe/erpnext --branch "$BRANCH_TO_CLONE" --resolve-deps
+bench get-app https://github.com/frappe/erpnext --branch "$BRANCH_TO_CLONE"
+
+bench get-app healthcare "${GITHUB_WORKSPACE}"
 bench setup requirements --dev
+CI=Yes bench build --app frappe &
```

Comment:
The main fix here is to install everything in python environment and then only start bench process.